context/app/static/js/components/savedLists/listTextFields/index.js

```diff
@@ -13,7 +13,8 @@ function TitleTextField({ handleChange, title }) {
       label="Title"
       fullWidth
       variant="outlined"
-      placeholder='Like “Spleen-Related Data” or “ATAC-seq Visualizations”'
+      // eslint-disable-next-line prettier/prettier
```

Comment:
Why the change?

Reply:
This is needed for one of the formatting changes you requested in an earlier PR.